Gemfile

```diff
@@ -46,6 +46,7 @@ gem 'webpacker', '~> 4.x'
 gem 'ed25519', '~> 1.2'
 gem 'bcrypt_pbkdf', '~> 1'
 gem 'kaminari'
+gem 'foreman'
```

Comment:
INFO: Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `foreman` should appear before `kaminari`.